fe/fe-core/src/main/java/org/apache/doris/job/extensions/mtmv/MTMVTask.java

```diff
@@ -62,7 +66,7 @@
 
 public class MTMVTask extends AbstractTask {
     private static final Logger LOG = LogManager.getLogger(MTMVTask.class);
-    public static final Long MAX_HISTORY_TASKS_NUM = 100L;
+    public static final Integer DEFAULT_REFRESH_PARTITION_NUM = 1;
```

Comment:
why Integer?

Reply:
List.subList() param is `int`,If using `long`, I need to convert it to `int`